cdh_prototype/wrappers/com_wrap.c

```diff
@@ -875,11 +867,12 @@ void uplink_handshake(uint32_t* cmd_packet_size) {
 	*/
 
 #if COM_ENABLE
-	size_t n = 0;
-	if(!(kLPUART_RxDataRegEmptyFlag & LPUART_GetStatusFlags(COM_RTOS_UART_HANDLE)) ) { //recv_buffer not empty
-		/* receive Transmission Primary Header & ACKNOWLEDGEMENT */
-		LPUART_RTOS_Receive(&COM_RTOS_UART_HANDLE, &uplink_recv_buffer, (uint32_t)(PRIMARY_HEADER_SIZE + ACK_SIZE), &n);
-	}
+//TODO: Debug this
+//	size_t n = 0;
+//	if(!(kLPUART_RxDataRegEmptyFlag & LPUART_GetStatusFlags(COM_RTOS_UART_HANDLE)) ) { //recv_buffer not empty
+//		/* receive Transmission Primary Header & ACKNOWLEDGEMENT */
+//		LPUART_RTOS_Receive(&COM_RTOS_UART_HANDLE, &uplink_recv_buffer, (uint32_t)(PRIMARY_HEADER_SIZE + ACK_SIZE), &n);
+//	}
```

Comment:
Should this be commented?

Reply:
Uncertain. This will be grabbing the header (containing the incoming HMAC hash) from ground station command packets. Requires testing, which requires a better understanding of how com works.

Reply:
Ok sounds good. then leaving it commented for now is probably fine